Makefile

```diff
@@ -11,9 +11,14 @@ lint: install
 	npx eslint --ignore-pattern=**/target --ignore-pattern=**/node_modules --ignore-pattern=**/dist --ignore-pattern=**/*.conf.js --ignore-pattern=**/webpack.config.js **/*.ts
 	mvn org.apache.maven.plugins:maven-checkstyle-plugin:check -Dverbose=true
 
+before: install
```

Comment:
I'd recommend adding the contents of this target to the earlier `install` target.

Reply:
Right now, when one of these targets is built, `npm install` is run on the top-level directory every time. Making your suggested change would cause `npm install` to be run in the `webapp` directory every time as well, which you suggested avoiding in #18.

If I make this change, do you want me to remove the `install` dependency on `dev`, `prod`, and `deploy`?

Reply:
removing the install deps makes sense, but it would still be cleaner if we moved the contents of the `before` target inside of this. I won't block on it, but it'd be cleaner for the `install` target to install node_modules for both dirs without the extra level of indirection of `npm run before`.

It is more awkward for the case of `build:dev` and `build:prod` to inline the commands in the makefile since they directly depend on the contents of its node_modules, and `npm` handles finding the binaries.